q2_fragment_insertion/plugin_setup.py

```diff
@@ -79,4 +74,70 @@
 )
 
 
+# text for readme, if we decide to put function 'classify_paths' back in:
```

Comment:
Can these comments just be deleted?

Reply:
you mean the whole block of out-commented code or just one specific line?

Reply:
@wasade another ping :-)